aya/src/obj/mod.rs

```diff
@@ -494,6 +494,38 @@ impl Object {
         Ok(())
     }
 
+    fn parse_map_section(
+        &mut self,
+        section: &Section,
+        symbols: Vec<Symbol>,
+    ) -> Result<(), ParseError> {
+        if symbols.is_empty() {
+            return Err(ParseError::NoSymbolsInMapSection {});
+        }
+        let chunk_size = if section.size % symbols.len() as u64 == 0 {
```

Comment:
Can the `.maps` section contain variable length `bpf_map_def`s? (we need to check libbpf). See `MINIMUM_MAP_SIZE` which we support to support `iproute2` style maps.

Reply:
`maps` can contain multiple map defs.... but they all MUST be the same size.
Here's the map counting/iteration logic and you can see the assumption in the comments.
https://github.com/libbpf/libbpf/blob/master/src/libbpf.c#L1968